### ISSUE TITLE
Can this be used to make official game?

### ISSUE THREAD

**Issue by Drakaroft** [Stary2001/godot#26]
Once (and if) one become official Nintendo third party developer, can send the files that export gives to Nintendo to make it an official game? (I know that there are other stuff requsted like trailer ecc), i'm saying if the file from exporter are fine to become an official game or it will work only on modded (homebrew) console.
Thanks, and sorry if the question may sound stupid, i don't know much of the Switch publishing ecosystem.

**Comment by Vadimledyaev** [comment]
Hey, I'd like to know the answer, too

**Comment by hippydave** [comment]
No, this is built against the homebrew toolchain, for an official game you would need Godot built against Nintendo's official toolchain. If you can't do that yourself, there are game porting companies listed on the Godot website

**Comment by mustafa-archived** [comment]
Sorry I'm late, this is a tut for Nintendo switch (if) you became an official dev https://www.reddit.com/r/godot/comments/nbnd50/how_to_godot_on_the_switch/
